docs/quick-start.md

```diff
@@ -16,11 +16,33 @@ Confirm that kubectl can connect to the cluster.
 kubectl cluster-info
 ```
 
+Install cert-manager using helm. Note that because you are using a GKE
+Autopilot cluster, you need to use this particular version with these specific
+cli arguments to make cert-manager work on your GKE Autopilot cluster. 
+
+```shell
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --version "v1.9.1" \
```

Comment:
Is there a documented reason why Autopilot needs this version?

Reply:
No, but I tested this script both with 1.9.1 and the latest version, 1.10.1. It worked with 1.9.1 and failed with 1.10.1. 

I don't think the problem with 1.10.1 is specific to GKE Autopilot though. I suspect that they changed the API, 
and as a result we will need to update our configuration YAML. I have not investigated this yet. I created issue
#162 to track this task.